scapy/contrib/nsh.py

```diff
@@ -59,31 +53,39 @@ class NSH(Packet):
     fields_desc = [
         BitField('Ver', 0, 2),
         BitField('OAM', 0, 1),
-        BitField('Critical', 0, 1),
-        BitField('Reserved', 0, 6),
-        BitFieldLenField('Len', None, 6,
-                         count_of='ContextHeaders',
+        BitField('Unused1', 0, 1),
+        BitField('TTL', 63, 6),
+        BitFieldLenField('Length', None, 6,
```

Comment:
Also please remain consistent in the choice of this name: `len` or `length` for both.